draft-tjjk-cared.md

```diff
@@ -327,6 +347,16 @@ scenarios where client authentication for encrypted DNS is limited
 to pre-existing relationships between the client and server, there
 should be no need for at-run-time intervention by a human user.
 
+### Resistance to replay attacks
+
+Today, there are some attempts to identify clients that involve use of client-
+specific DoH templates or DoT hostnames, addition of magic strings to requests,
+and other mechanisms to enable proprietary experiences. It is important that
+recommendations for client authentication are restricted to mechanisms that
+would require compromise of secrets versus configurations which may not enjoy
```

Comment:
This sentence is a tough read for me, it took me a few times. i think it's saying that you shouldn't use an authentication that allows unprotected secrets/keys ?

Reply:
Yes, indeed that was my intent. Gimme a couple minutes, I'll rephrase (you're right, this is awkward)

Reply:
How about the new text?